src/Microsoft.Health.Fhir.Core/Features/Search/Converters/FhirTypedElementToSearchValueConverterManager.cs

```diff
@@ -36,5 +41,36 @@ public bool TryGetConverter(string fhirType, Type searchValueType, out ITypedEle
 
             return _converterDictionary.TryGetValue((fhirType, searchValueType), out converter);
         }
+
+        internal class ExtensionConverter : ITypedElementToSearchValueConverter
+        {
+            private readonly List<ITypedElementToSearchValueConverter> _searchValueTypeConverters;
+
+            public ExtensionConverter(Type searchValueSearchValueType, List<ITypedElementToSearchValueConverter> searchValueTypeConverters)
+            {
+                EnsureArg.IsNotNull(searchValueSearchValueType, nameof(searchValueSearchValueType));
+                EnsureArg.IsNotNull(searchValueTypeConverters, nameof(searchValueTypeConverters));
+                EnsureArg.HasItems(searchValueTypeConverters, nameof(searchValueTypeConverters));
+
+                _searchValueTypeConverters = searchValueTypeConverters;
+                SearchValueType = searchValueSearchValueType;
+            }
+
+            public IReadOnlyList<string> FhirTypes => new List<string> { "Extension" };
+
+            public Type SearchValueType { get; }
+
+            public IEnumerable<ISearchValue> ConvertTo(ITypedElement value)
+            {
+                if (value == null)
+                {
+                    return new List<ISearchValue>();
```

Comment:
>List<ISearchValue>() [](http://example.com/codeflow?start=31&length=20)

nit: [https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.empty?view=net-5.0](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.empty?view=net-5.0)

Reply:
Ah yes that is nicer, thanks. Fixed in a7c8f99.